program/rust/src/tests/test_time_machine.txt

```diff
@@ -0,0 +1,30 @@
+test initialize for:
```

Comment:
Maybe delete this file?

Reply:
I will use it when I write tests feel free to ignore it meanwhile

Reply:
you are not gonna merge without tests anyway